src/controller/java/zap-generated/CHIPReadCallbacks.cpp

```diff
@@ -35184,7 +35173,7 @@ void CHIPUnitTestingNullableCharStringAttributeCallback::CallbackFn(void * conte
     }
     else
     {
-        javaValue = env->NewStringUTF(std::string(value.Value().data(), value.Value().size()).c_str());
+        chip::JniReferences::GetInstance().CharToStringUTF(value.Value(), javaValue);
```

Comment:
Should use the return error.

Reply:
When doing TLV Decode, I think it is best to implement it in the same way as returning to null if an error occurs.
We added setting null to the outString variable in CharToStringUTF and wrapped it with LogErrorOnFailure to add log when it is an error.
(LogErrorOnFailure is Marco that only outputs Log when Error.)